modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -136,15 +148,65 @@ Properties set: 8
 Labels added: 4
 ----
 
+[NOTE]
+====
+You can use the hexadecimal representation of the unicode character for the field delimiter if you prepend `{backslash}u`.
+Write the encoding with four digits, for example, `{backslash}u003B` is equivalent to `;` (semicolon).
+====
+
 
-[[load-csv-import-data-from-a-csv-file-containing-headers]]
-== Import data from a CSV file containing headers
-When your CSV file has headers, you can view each row in the file as a map instead of as an array of strings.
+[[load-csv-character-escaping-and-quotes]]
+=== Character escaping and quotes
 
-.artists-with-headers.csv
-[source, csv, role="noheaders", filename="artists-with-headers.csv"]
+If the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings[configuration setting] `dbms.import.csv.legacy_quote_escaping` is set to `true` (the default value), `\` is used as the escape character: `"The {backslash}"Symbol{backslash}""`.
```

Comment:
hm... the link does not target the actual setting, but the general configuration settings section instead.
I think it's better to remove the link as this might be confusing